pom.xml

```diff
@@ -50,58 +50,75 @@
         <target>all</target>
     </properties>
 
-    <build>
-        <plugins>
-            <plugin>
-                <artifactId>maven-antrun-plugin</artifactId>
-                <version>1.8</version>
-                <executions>
-                    <execution>
-                        <id>process-build-all</id>
-                        <phase>compile</phase>
+
+    <profiles>
+
+        <profile>
+            <id>no-go-client</id>
+        </profile>
+
+        <profile>
+            <id>go-client</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
Will be skipped when activating any profile, not just `no-go-client`. For example when testing (`-PIntegration`). Could use a property activator instead - see [deployTo](https://github.com/apache/brooklyn-server/blob/master/parent/pom.xml#L1779) example.